CHANGES.md

```diff
@@ -52,6 +52,12 @@
 
 - Set version in `META` and `dune-package` files to the one read from
   the vcs when no other version is available (#2224, @diml)
+  
+- Add a variable %{target} to be used in situations where the context
```

Comment:
```suggestion
- Add a variable `%{target}` to be used in situations where the context
```

and the next line as well. The square brackets should be replaced by backquotes in the last two lines to follow the markdown syntax

Reply:
Thanks. Fixed.